### PR TITLE
Add more CI test platforms

### PR DESCRIPTION
**Pull request by nyurik**
Ensure this action would pass on all available GitHub runners

### CODE REVIEW

**Comment by ikalnytskyi** [comment]
Hey @nyurik,

Thank you for the patch and sorry for being unresponsive that long. The patch makes sense to me and I'd be happy to accept it. Let us first figure out how to proceed with broken tests now. See https://github.com/ikalnytskyi/action-setup-postgres/issues/38#issuecomment-2218839834 for details.

**Comment by ikalnytskyi** [comment]
Hey @nyurik,

May I ask you to rebase your change? I believe the broken test is fixed in the master branch.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,13 +18,18 @@ jobs:
     strategy:
       matrix:
         os:
+          - ubuntu-latest
```
Comment (ikalnytskyi) [inline .github/workflows/ci.yml:21]: `*-latest` runners are tested in the `parametrized` job. So maybe we can save some CPU cycles by not running them here.

Comment (nyurik) [reply .github/workflows/ci.yml]: thx, done

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -18,13 +18,15 @@ jobs:
...
         os:
-          - ubuntu-22.04
           - ubuntu-20.04
-          - windows-2022
-          - windows-2019
-          - macos-14
-          - macos-13
+          - ubuntu-22.04
+          - ubuntu-24.04
+          - macos-11
```
Comment (ikalnytskyi) [inline .github/workflows/ci.yml:24]: I don't think `macos-11` exist, and this is why the check could not finish. It looks like the only missing platform is `ubuntu-24.04`, according to the GitHub docs:

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

So I think this patch could be a 1 line change, after all. :thinking:

Comment (nyurik) [reply .github/workflows/ci.yml:24]: ah, they removed 11 i guess. will fix

**Pull request merged**